views/includes/header.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(options) {
           </div>
           <button type="submit">
             Search
-            ${require('./svg')({svgBasename: 'magnify'})}
+            ${require('./svg.js')({svgBasename: 'magnify'})}
```

Comment:
As described [in the API Reference](https://nodejs.org/api/modules.html#modules_all_together), the exact filename is tried first, so we actually save one filesystem call. Apart from that, it makes it more readable because our own files look more distinct from packages (especially ones with slashes like `vue/dist/vue` or `uuid/v4`).